pandas/io/tests/test_clipboard.py

```diff
@@ -113,3 +113,12 @@ def test_read_clipboard_infer_excel(self):
         exp = pd.read_clipboard()
 
         tm.assert_frame_equal(res, exp)
+
+    # unicode round trip test for GH 13747    
+    def test_round_trip_frame_unicode(self):
+        sep = ','
+        df = pd.DataFrame({'a':['µasd','Ωœ∑´'], 'b':['øπ∆˚¬','œ∑´®']})
```

Comment:
Placing it in `setUpClass` just allows to run all tests automatically. Whether it's similar to `en espanol` is another thing (Chinese or cyrillic characters might be less similar but it's probably ok).